content/posts/podcast.md

```diff
@@ -17,6 +17,7 @@ In deze podcast interviewen Mark en Tim interessante mensen uit onze community o
 
 Heb je een vraag naar aanleiding van de podcast, heb je zelf een leuk onderwerp of een gast die goed zou passen? Laat het ons dan weten door een mail te sturen naar [podcast@nuts.nl](mailto:podcast@nuts.nl).
 
-Luister hier meteen de nieuwste aflevering via Spotify:
+Luister hier meteen de nieuwste aflevering via Spotify van twee concurrenten die beiden het grote nut zien :wink: van de Nuts-community en het Nuts-gedachtegoed 
```

Comment:
Dit suggereert dat dit de nieuwste aflevering is van onze eigen podcast, maar dit is een andere podcast.

Reply:
Ik heb er als titel van gemaakt Going Nuts Podcast. 
Podcast Geheelmeesters is nu apart met eigen titel.